ui/components/ConfirmationModal.js

```diff
@@ -284,18 +284,24 @@ function ConfirmationMsg(props) {
             <Tab
               data-cy="validate-btn-modal"
               className={classes.tab}
-              label={<div style={{ display : "flex" }}> <DoneIcon style={{ margin : "2px" }}  fontSize="small"/><span className={classes.tabLabel}>Validate</span> </div>
+              label={<div style={{ display : "flex" }}
+                onClick={(e) => handleTabValChange(e,0)}
+              > <DoneIcon style={{ margin : "2px" }}  fontSize="small"/><span className={classes.tabLabel}>Validate</span> </div>
               }
             />}
             <Tab
               data-cy="Undeploy-btn-modal"
               className={classes.tab}
-              label={<div style={{ display : "flex" }}> <div style={{ margin : "2px" }}> <UndeployIcon fill="rgba(0, 0, 0, 0.54)" width="20" height="20"/> </div> <span className={classes.tabLabel}>Undeploy</span> </div>}
+              label={<div style={{ display : "flex" }}
+                onClick={(e) => handleTabValChange(e,1)}
```

Comment:
Use event instead of `e` or another relevant term.

Reply:
Done